community/sc-im/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+cd src
+
+make
+make DESTDIR="$1" PREFIX=/usr install
```

Comment:
Remove the `cd` and use `-C src` for both `make` commands.

Reply:
Done.